src/test/libslic3r/test_print.cpp

```diff
@@ -138,8 +138,18 @@ SCENARIO("Print: Brim generation") {
             config->set("first_layer_extrusion_width", 0.5);
             auto print {Slic3r::Test::init_print({m}, model, config)};
             print->make_brim();
-            THEN("Brim Extrusion collection has 12 loops in it") {
-                REQUIRE(print->brim.items_count() == 12);
+            double nbLoops = 6.0 / print->brim_flow().spacing();
```

Comment:
Why the change here for the extrusion collection?

Reply:
I made it that way because I wanted it to check if it makes the correct number of loop and not checking if the spacing is exactly what we tough about it (the layer heigh isn't set, btw)
I should had changes other tests also, i think that only this one bother me at the time because it was so large, a little difference in spacing can change the line count.